notebook/static/notebook/js/notificationarea.js

```diff
@@ -399,12 +399,14 @@ define([
                 }, {'title':'Javascript enabled for notebook display'});
                 // don't allow 'Trusted' button to be clicked
                 $(tnw.selector).attr('disabled', true)
+                $(tnw.selector).attr('role', 'button')
```

Comment:
Is it still appropriate to use the 'button' role when it's the 'trusted' notification which can't be clicked?

Reply:
Initial thoughts were that even if the element was not clickable (disabled), it is still a button. If the notification was created as a button, it's role would not change when it is disabled. The role attribute has been removed from the Trusted notification. Since the element is a span and is disabled there is no difference based on the presence of the attribute.